bluebrain/repo-bluebrain/packages/py-fz-td-recipe/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyFzTdRecipe(PythonPackage):
+    """Python package to read and modify the definitions and parameters used in circuit
+    building."""
+
+    homepage = "https://bbpteam.epfl.ch/documentation/projects/fz-td-recipe"
+    git      = "git@bbpgitlab.epfl.ch:hpc/circuit-building/fz-td-recipe.git"
+
+    version('develop', branch='master')
+    version('0.1.0.dev0', tag='fz-td-recipe-v0.1.0.dev0')
```

Comment:
`nitpick`: unusual tag naming ?

Reply:
NSE style :)

Reply:
I used only the NSE pipelines for this, as it is a pure Python project. This is what their tagging does 🤷

Reply:
Historical reasons; we supported, at one point, having multiple python packages within the same repo (think monorepo), so that python package name needed to be in the tag, to distinguish different releases.